src/event_loop.rs

```diff
@@ -136,7 +136,7 @@ impl EventLoop<()> {
     /// ## Platform-specific
     ///
     /// - **iOS:** Can only be called on the main thread.
-    pub fn new() -> EventLoop<()> {
+    pub fn new() -> Result<EventLoop<()>, String> {
```

Comment:
Check doc on this method

Reply:
I am guessing you mean change the docs for this one